svc/pkg/middleware/auth.go

```diff
@@ -54,6 +64,27 @@ func (a auth) VerifyUser() gin.HandlerFunc {
 			c.AbortWithError(401, err)
 			return
 		}
+
+		// Userの存在チェック
+		if result.UserID == "" {
+			c.AbortWithStatusJSON(500, "user_id is null")
+			return
+		}
+		_, err = a.q.User.WithContext(c).Where(gQuery.User.FirebaseUID.Eq(result.UserID)).First()
+		if err != nil {
+			if errors.Is(err, exception.ErrNotFound) {
```

Comment:
domain導入していた時は `exception.ErrNotFound` でハンドリングをしていたけれど、今回は Gorm のエラーに対応しないといけなさそう..!
```suggestion
			if errors.Is(err, gorm.ErrRecordNotFound) {
```

Reply:
こちらで対応！
[0c40075](https://github.com/openhacku-team-a/a-project-backend/pull/26/commits/0c4007543a72babaa790579708498a08fc1b70a8)